dhall-to-cabal.cabal

```diff
@@ -1,4 +1,13 @@
 name: dhall-to-cabal
+-- * * * * * * * * * * * * WARNING * * * * * * * * * * * *
+-- This file has been AUTO-GENERATED by dhall-to-cabal.
+--
+-- Do not edit it by hand, because your changes will be over-written!
+--
+-- Instead, edit the source Dhall file, namely
+-- 'dhall-to-cabal.dhall', and re-generate this file by running
+-- 'dhall-to-cabal -- dhall-to-cabal.dhall'.
```

Comment:
Should this include the shell redirection? `dhall-to-cabal` doesn't write files.

Reply:
The issue is we don't necessarily know the filename, since the input Dhall file could have an arbitrary name, so I was hoping that the user would see the output and get the idea. I'll add a note about redirecting the output.

Reply:
Oh, we don't know the file name of the `.dhall` file itself. I didn't think about that! That makes sense. In that case, I think this is OK as is, but maybe being clear that you have to save the output would help. E.g., `dhall-to-cabal -- dhall-to-cabal.dhall > THIS_FILE( where THIS_FILE is the name of this file.)`